api.planx.uk/modules/webhooks/_old/lowcalSessionEvents.test.ts

```diff
@@ -1,10 +1,10 @@
 import supertest from "supertest";
-import app from "../server";
-import { createScheduledEvent } from "../hasura/metadata";
+import app from "../../../server";
```

Comment:
If the new API structure leads to a lot of long/messy relative import paths we could look at resolving this with path aliases -  https://blog.logrocket.com/using-path-aliases-cleaner-react-typescript-imports/